docs/notebooks/recovering_from_errors.pct.py

```diff
@@ -76,7 +76,7 @@ def __call__(self, x):
 
 num_steps = 15
 result, history = bo.optimize(
-    num_steps, initial_data, model, acquisition_rule
+    num_steps, initial_data, model, acquisition_rule, None
```

Comment:
This is required to get a state-based return type that's consistent with the one on line 108. Also it's more explicit as we're using TrustRegion.